systemtests/src/test/java/io/enmasse/systemtest/TestBase.java

```diff
@@ -71,12 +77,18 @@ public void setup() throws Exception {
         addressSpaceList = new ArrayList<>();
         amqpClientFactory = new AmqpClientFactory(openShift, environment, null, username, password);
         mqttClientFactory = new MqttClientFactory(openShift, environment, null, username, password);
+        managementAmqpClientFactory = new AmqpClientFactory(openShift,
+                environment,
+                null,
+                managementCredentials.getUsername(),
+                managementCredentials.getPassword());
```

Comment:
Not sure why you'd create another factory here. The username/password is just about defaults. You can create clients where you set the username/password explicitly to something different instead.

Reply:
that's true, thank you